.github/ISSUE_TEMPLATE/4_PLIP_task.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "📋 PLIP task"
+about: 'Plone Improvement Proposal (PLIP) tasks are sub-issues of the umbrella PLIP.'
+title: ''
+labels: '06 type: plip task'
+type: 'Feature'
+assignees: ''
+projects: 'plone/47'
+---
+
+> [!IMPORTANT]
+> If you are not a member of the Volto Team or Developers Team in the Plone GitHub organization, then do not work on or comment on this issue.
+
+**Describe the PLIP task**
+
+A clear and concise description of what you want to happen, even if it is already in the PLIP. A conversation about this PLIP task should continue and extend from the PLIP.
+
+**Link to the parent PLIP**
```

Comment:
The link to the parent appears already at the top.

Reply:
Yes, however, do you really want a couple dozen conversations for each PLIP task to be in the PLIP? Yikes!

Reply:
Miscommunication here. If you create a sub-issue from a PLIP, after you submit the new PLIP task, then a link to the parent PLIP appears above the created issue. But if you don't do that, then no such link appears. You can still go back and add an existing issue as a sub-issue to get that afterward. In any case, it's new and easily overlooked in the UI, so probably best to link it in the issue description.